pilot/pkg/model/context.go

```diff
@@ -443,11 +443,6 @@ type BootstrapNodeMetadata struct {
 	// of the workload instance (ex: k8s deployment for a k8s pod).
 	Owner string `json:"OWNER,omitempty"`
 
-	StatsInclusionPrefixes string `json:"sidecar.istio.io/statsInclusionPrefixes,omitempty"`
-	StatsInclusionRegexps  string `json:"sidecar.istio.io/statsInclusionRegexps,omitempty"`
-	StatsInclusionSuffixes string `json:"sidecar.istio.io/statsInclusionSuffixes,omitempty"`
-	ExtraStatTags          string `json:"sidecar.istio.io/extraStatTags,omitempty"`
-
```

Comment:
I think I would prefer that bootstrap node metadata had annotations and labels fields to satisfy the purity of the bootstrap derivation. The way it is now, the function reads files so it is hard to port this function into another place to compute.

Reply:
We can do the same as labels: https://github.com/istio/istio/blob/3d7671c491d678aff701eb9a0cb48006494ca391/pkg/bootstrap/config.go#L481 and have a dedicated Annotations section.